Makefile

```diff
@@ -46,13 +46,15 @@ up: init
 	docker compose up -d
 	sleep 15 # waiting for pmm server to be ready :(
 	docker compose exec pmm-client pmm-agent setup || true
+	docker exec pmm-server sed -i 's#<!-- <listen_host>0.0.0.0</listen_host> -->#<listen_host>0.0.0.0</listen_host>#g' /etc/clickhouse-server/config.xml
+	docker exec pmm-server supervisorctl restart clickhouse
```

Comment:
This is broken when .env has custom name for pmm-server container.

Reply:
https://github.com/percona/pmm-dump/pull/99/commits/311d553e2d20150e2fdfb306ad837b7350f7aaba